src/Akeneo/Pim/Enrichment/Bundle/PdfGeneration/Builder/DompdfBuilder.php

```diff
@@ -24,12 +25,15 @@ class DompdfBuilder implements PdfBuilderInterface
      */
     protected $dompdf;
 
+    protected HtmlFormatter $arabicHtmlFormatter;
+
     private string $publicDir;
 
-    public function __construct(string $rootDir, $publicDir)
+    public function __construct(string $rootDir, $publicDir, HtmlFormatter $arabicHtmlFormatter)
```

Comment:
```suggestion
    public function __construct(string $rootDir, $publicDir, HtmlFormatter $arabicHtmlFormatter = null)
```

Adding an argument is a BC break. We need to set it optional

Reply:
:+1: ok, it's noted, thx